pkg/apis/eventing/v1alpha1/broker_lifecycle.go

```diff
@@ -47,6 +47,11 @@ func (bs *BrokerStatus) GetCondition(t apis.ConditionType) *apis.Condition {
 	return brokerCondSet.Manage(bs).GetCondition(t)
 }
 
+// GetTopLevelCondition returns the top level Condition.
+func (bs *BrokerStatus) GetTopLevelCondition() *apis.Condition {
```

Comment:
This change needs to happen in Serving, Sample-Source and Sample-Controller now too right?

Reply:
Yep. I think this change needs to happen in Serving and Sample-Source. But it won't break anything if people don't use it. People can also use `GetCondition(t apis.ConditionType)` to get the top level condition. `GetTopLevelCondition` just provides a more convenient way to achieve it. What do you mean by "Sample-Controller"?